infrastructure/backend.tf

```diff
@@ -17,7 +17,7 @@
 terraform {
   backend "s3" {
     bucket         = "terraform-state-xc3" // S3 bucket for terraform state management
-    key            = "xc3/xc3.tfstate"     // S3 object key for terraform state file to maintain history of resources
+    key            = "xc3/xc3.tfstate"        // S3 object key for terraform state file to maintain history of resources
```

Comment:
```suggestion
    key            = "xc3/xc3.tfstate"     // Specifies the S3 object key for storing the Terraform state file
```

Reply:
Added in 60a640a535551a9fe00d29030e459e9a3e59bfec.

Yes @faheem-khau9 this has been tested through pipeline
![image](https://github.com/XgridInc/xc3/assets/122358742/e2df2391-ce41-4b98-a1da-9dae5e2c8dc9)